tests/performance/README.md

```diff
@@ -0,0 +1,37 @@
+## Performance testing usage guide
+
+
+### Locust
+
+- Install locust, `make build` should do it, but if it's not in the pipfile then `pipenv install locustio`
+- `pipenv run locust -f /path/to/locustfile.py` - this will start a server on `http://localhost:8089` that uses the specified locustfile
+-  Go to the above url, specify the number of users and the spawn rate
+
+#### create-messages script guide
+To create a number of messages quickly, there is a create-messages.py script.
```

Comment:
If this needs to be done first it might be handy to write a `run_perforamce_test.sh` which gets everything set up so that it can be run (steps below etc)

Reply:
Whatcha mean sorry?  The steps to get the right data is manual so I can't script it up, and getting into a stage where it can be automated will take a bunch more work that probably isn't worth it in light of all the other stuff that we need to get done.

Reply:
Yeah, I think I was tired when I wrote this, I think my idea behind it was more to do with automation but there's no need for that now, but if we could get automation in the future it would give us a lot of confidence making changes. It works now so it's not a problem 👍